boxes/npx.js

```diff
@@ -43,6 +58,70 @@ function updatePathEnvVar() {
   info(`Added ${targetDir} to PATH in ${shellProfile}.`);
 }
 
+export function prettyPrintNargoToml(config) {
+  const withoutDependencies = Object.fromEntries(
+    Object.entries(config).filter(([key]) => key !== "dependencies"),
+  );
+
+  const partialToml = stringify(withoutDependencies);
+  const dependenciesToml = Object.entries(config.dependencies).map(
+    ([name, dep]) => {
+      const depToml = stringify.value(dep);
+      return `${name} = ${depToml}`;
+    },
+  );
+
+  return (
+    partialToml + "\n[dependencies]\n" + dependenciesToml.join("\n") + "\n"
+  );
+}
+
+async function replacePaths(rootDir) {
+  const files = fs.readdirSync(path.resolve(".", rootDir), {
+    withFileTypes: true,
+  });
+
+  files.forEach((file) => {
+    const filePath = path.join(rootDir, file.name);
+    // console.log(filePath);
+    if (file.isDirectory()) {
+      replacePaths(filePath); // Recursively search subdirectories
+    } else if (file.name === "Nargo.toml") {
+      let content = parse(fs.readFileSync(filePath, "utf8"));
+
+      try {
+        Object.keys(content.dependencies).forEach((dep) => {
+          const directory = content.dependencies[dep].path.replace(
+            "../../../../",
+            "",
+          );
```

Comment:
I'm worried that reorganizing packages locations may break things again. Should we instead replace `/^(..\/)+/` so we capture whatever many parent directories we need?

Reply:
Yep, I copied this as it was in unbox but that was the reason why the `unbox` command was broken actually! So I'll fix it